test/integration/run_envoy_test.sh

```diff
@@ -19,10 +19,6 @@ function expect_fail_with_error() {
 start_test Launching envoy with a bogus command line flag.
 expect_fail_with_error "PARSE ERROR: Argument: --bogus-flag" --bogus-flag
 
-start_test Launching envoy without --config-path or --config-yaml fails.
```

Comment:
This test should be retained, even if with a slightly different error message per above.

Reply:
Done.